docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -92,7 +92,8 @@ Constructors
 
 .. function:: ST_Point(double, double) -> Point
 
-    Returns a geometry type point object with the given coordinate values.
+    Returns a geometry type point object with the given coordinate values in the
+    form (longitude, latitude).
```

Comment:
This is actually incorrect. This function returns a point on a plane. The choice of whether to use it to represent a longitude/latitude coordinate is up to the user and context dependent.

Reply:
@sheajamba can update this. Is there some information we can add on SphericalGeography functions and what we support in that context .. doesnt seem like we have much in the current docs..